hypershift-operator/main.go

```diff
@@ -231,7 +231,14 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 		HypershiftOperatorImage:       operatorImage,
 		ReleaseProvider: &releaseinfo.RegistryMirrorProviderDecorator{
 			Delegate: &releaseinfo.CachedProvider{
-				Inner: &releaseinfo.RegistryClientProvider{},
+				Inner: &releaseinfo.StaticProviderDecorator{
+					Delegate: &releaseinfo.RegistryClientProvider{},
+					ComponentImages: map[string]string{
+						// TODO: Remove this override when we update CAPA to match the master branch of openshift/cluster-api-provider-aws
+						// Currently, this image points to CAPA in OCP release 4.12.0-ec.5
+						"aws-cluster-api-controllers": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0fe3b95dad471e6840e8f38624fcd0ee6cbc92503460fe7f3503e640e13402d3",
```

Comment:
Would it be possible to use a multi-arch image here rather than an x86 only image?

Reply:
Hey @bryan-cox yup, that makes sense, will update.

Reply:
Updated